contracts/0.4.24/template/LidoTemplate.sol

```diff
@@ -614,6 +619,20 @@ contract LidoTemplate is IsContract {
         acl.createPermission(_state.stakingRouter, _state.operators, _state.operators.STAKING_ROUTER_ROLE(), voting);
         acl.createPermission(_state.agent, _state.operators, _state.operators.MANAGE_NODE_OPERATOR_ROLE(), voting);
 
+        if (deployState.sdvt != address(0)) {
```

Comment:
Not sure about this optionality branching (see my comments with the 'optionality' word)

Reply:
Same here. Agree to remove optionality if there will be no deploys without SDVT in future.